### PR TITLE
Loosing focus in message list when remove a message (Suppr)

### PR DESCRIPTION
**Pull request by misstick**
Actually `_fixCurrentMessage ` should do that stuff but it doesnt work.

When a message is deleted : 
 - [x] next message in the list should be focus,
 - [x] URL should be updated.

When a user navigate with keyboard into messageList : 
 - [x] next message should be focus when I press on `top` key,
 - [x] previous message should be focus when I press on `bottom` key.

### CODE REVIEW

**Comment by frankrousseau** [comment]
Could you add comments telling what your function do and why you chose do it that way (if needed).

**Comment by misstick** [comment]
@frankrousseau : I thought function names are the most explicit here, didnt it?

**Comment by frankrousseau** [comment]
Yep, but you can explain how the function works and why you choosed to do it that way.

File: client/app/utils/api_utils.coffee [hunk 1]
```diff
@@ -95,21 +93,21 @@ module.exports =
...
     messageNavigate: (direction, inConv) ->
-        if not onMessageList()
-            return
+        return unless onMessageList()
         conv = inConv and SettingsStore.get('displayConversation') and
             SettingsStore.get('displayPreview')
+
         if direction is 'prev'
-            next = MessageStore.getPreviousMessage conv
+            next = MessageStore.getNextConversation()
```
Comment (aenario) [inline client/app/utils/api_utils.coffee]: this seems reverted.

**Comment by frankrousseau** [comment]
It's ok for me. @aenario Do you think we can merge it?

**Comment by misstick** [comment]
@frankrousseau : I'm adding comments on store, to be more explicit as you asked before

**Comment by misstick** [comment]
@frankrousseau, @aenario : it's OK for me

**Pull request merged**